source/tools/testConsistency/ClientApp/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "clientapp",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "jest"
+  },
+  "author": "",
+  "license": "ISC",
```

Comment:
these fields should align with the values we use in our public nodejs packages:

https://github.com/microsoft/AdaptiveCards/blob/9b27ee8d79f0010ec246ec736e5722bb54f436f8/source/nodejs/adaptivecards/package.json#L5-L6